ui/tests/acceptance/redirect-to-test.js

```diff
@@ -55,7 +55,7 @@ module('Acceptance | redirect_to query param functionality', function (hooks) {
     localStorage.clear();
   });
   test('redirect to a route after authentication', async function (assert) {
-    const url = '/vault/secrets/secret/create';
+    const url = '/vault/secrets/secret/kv/create';
```

Comment:
Should this be left at the old route to ensure the redirect in the `beforeModel` hook is hit?

Reply:
I had to double check this, but this URL is both the start url and finish url. What this test is checking is that it encoded the url and then after the redirect when to the same route.

Reply:
The URL does not change. It's the same one used in before and after—it's not testing a true redirect.

Reply:
The fact this test fails without me updating it tells me the bug fix is working... I know this is hard to visualize, apologies.

Reply:
That makes sense - thanks for explaining!